### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by AlwinEsch**
The following changes improve the client creation class and all XBMC parts are renamed to KODI and related URL's fixed.

Further is the README a bit updated with build steps on newest VDR Version, due to not from me know VDR Version which bring independent build I has set it to 2.2.0.

Also several other small parts.

### CODE REVIEW

**Comment by AlwinEsch** [comment]
OK to merge in?

File: HISTORY [hunk 1]
```diff
@@ -1,6 +1,13 @@
 VDR Plugin 'vnsiserver' Revision History
 ----------------------------------------
 
+2015-04-06: Version 1.3.1
```
Comment (FernetMenta) [inline HISTORY]: no need to bump version, we have not released 1.3.0 yet
https://github.com/FernetMenta/vdr-plugin-vnsiserver/releases

Comment (AlwinEsch) [inline HISTORY]: OK thanks :-)

File: README [hunk 2]
```diff
@@ -27,9 +27,21 @@ Get VDR sources.
    $ cd <path to vdr>/PLUGINS/src
    $ git clone https://github.com/FernetMenta/vdr-plugin-vnsiserver
    $ ln -s vdr-plugin-vnsiserver vnsiserver
-   
-   -----------------------------------------------------------------------------
+
+On newer VDR Versions (2.2.0) plugin can be builded and installed alone, use:
```
Comment (FernetMenta) [inline README]: builded -> built

**Comment by AlwinEsch** [comment]
Is updated.

**Comment by FernetMenta** [comment]
thanks, merge away

**Pull request merged**